### PR TITLE
Esperanto localization

### PR DESCRIPTION
**Pull request by n1rwana**

### CODE REVIEW

File: locales/eo.strings [hunk 1]
```diff
@@ -0,0 +1,587 @@
+"__locale" = "eo_EO.utf8;eo_EO.UTF-8;Esp";
```
Comment (veselcraft) [inline locales/eo.strings]: `eo.utf8` is correct

File: locales/eo.strings [hunk 2]
```diff
@@ -0,0 +1,587 @@
...
+"__WinEncoding" = "Windows-1251";
```
Comment (veselcraft) [inline locales/eo.strings]: wtf??

File: locales/eo.strings [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+"edit_page" = "Redakti paĝon";
+"edit_group" = "Redakti grupon";
+"change_status" = "ŝanĝi statuson";
+"name" = "Nomo";
+"surname" = "Familia nomo";
+"gender" = "Sekso";
+"male" = "vira";
+"female" = "ino";
+"description" = "Priskribo";
+"save" = "Savi";
```
Comment (veselcraft) [inline locales/eo.strings]: `konservi` is correct

**Comment by maksalees** [comment]
Not all strings have been translated in this translation. Can you translate in full? Thank you.

File: locales/eo.strings [hunk 4]
```diff
@@ -0,0 +1,587 @@
...
+"information" = "Informo";
+"status" = "Statuso";
+"no_information_provided" = "Neniu informo provizis.";
+"deceased_person" = "Forpasinta persono";
+"none" = "neniu";
+"send" = "Sendi";
+
+"years_zero" = "0 jaro malnova";
+"years_one" = "1 jaro malnova";
+"years_other" = "$1 jaroj malnova";
```
Comment (veselcraft) [inline locales/eo.strings]: just `jara` is ok

File: locales/eo.strings [hunk 5]
```diff
@@ -0,0 +1,587 @@
...
+"politViews" = "Politikaj vidoj";
+
+"politViews_0" = "Ne elektita";
+"politViews_1" = "Indiferenta";
+"politViews_2" = "Komunisto";
+"politViews_3" = "Socialisto";
+"politViews_4" = "Moderulo";
+"politViews_5" = "Liberalulo";
+"politViews_6" = "Konservativa";
+"politViews_7" = "Monarchical";
```
Comment (veselcraft) [inline locales/eo.strings]: `monarĥo`

File: locales/eo.strings [hunk 6]
```diff
@@ -0,0 +1,587 @@
...
+"politViews_9" = "Libervolisma";
+
+"contact_information" = "Contact information";
+
+"email" = "Email";
+"phone" = "Phone";
+"telegram" = "Telegram";
+"personal_website" = "Personal website";
+"city" = "City";
+"address" = "Address";
```
Comment (veselcraft) [inline locales/eo.strings]: why this wasn't translated?

File: locales/eo.strings [hunk 7]
```diff
@@ -0,0 +1,587 @@
...
+"add_signature" = "Aldonu subskribon";
+/* ^ can be translated as "author's signature". ^ */
+"contains_nsfw" = "Enhavas NSFW-enhavon";
+"nsfw_warning" = "Ĉi tiu afiŝo povas havi NSFW-enhavon";
+"report" = "Raporto";
+"attach_photo" = "Aligu foton";
+"no_posts_abstract" = "Neniu skribis ion ĉi tie... Ĝis nun.";
+"attach_no_longer_available" = "Ĉi tiu aldonaĵo ne plu disponeblas.";
+"open_post" = "Open post";
+"version_incompatibility" = "Ĉi tiu aldonaĵo ne povis esti montrata. Verŝajne la datumbazo estas nekongrua kun la nuna versio de OpenVK.";
```
Comment (veselcraft) [inline locales/eo.strings]: `de la`

File: locales/eo.strings [hunk 8]
```diff
@@ -0,0 +1,587 @@
...
+/* Albums */
+
+"create" = "Krei";
+"albums" = "Albumoj";
+"create_album" = "Krei albumon";
+"edit_album" = "Redakti albumon";
+"creating_album" = "Kreante albumon";
+"upload_photo" = "Alŝutu foton";
+"photo" = "Foto";
+"upload_button" = "Alŝutu";
```
Comment (veselcraft) [inline locales/eo.strings]: `Alŝuti`

File: locales/eo.strings [hunk 9]
```diff
@@ -0,0 +1,587 @@
...
+
+"notes_zero" = "Neniuj notoj";
+"notes_one" = "$1 noto";
+"notes_other" = "$1 notoj";
+
+/* Menus */
+
+/* Note that is string need to fit into the "My Page" link */
+
+"edit_button" = "redakti";
```
Comment (veselcraft) [inline locales/eo.strings]: check 263 line

File: locales/eo.strings [hunk 10]
```diff
@@ -0,0 +1,587 @@
...
+"login_as" = "Ensalutu kiel $1";
+"manage_user_action" = "Administri uzanton";
+"ban_user_action" = "Uzanto malpermeso";
+"warn_user_action" = "Averti uzanto";
+
+/* Paginator (deprecated) */
+
+"paginator_back" = "Reen";
+"paginator_page" = "Paĝo $1";
+"paginator_next" = "Venonta";
```
Comment (veselcraft) [inline locales/eo.strings]: not needed cos deprecated

File: locales/list.yml [hunk 11]
```diff
@@ -59,9 +59,8 @@ list:
...
-  - code: "ru_old"
-    flag: "ru_old"
-    name: "Pre-revolutionary"
-    native_name: "Дореволюцiонный"
-    author: "Mikitko Wiśniewski (rudzik8)"
-
+  - code: "eo"
+    flag: "eo"
+    name: "Esperanto"
+    native_name: "Эсперанто"
```
Comment (veselcraft) [inline locales/list.yml]: native name is the native name of language (for example the native name of kazakh language is Қазақша)

**Review by veselcraft — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/22993408/145690527-d743a294-db87-4e43-bd4f-4fc8e9f9f58c.mp4

**Comment by maksalees** [comment]
~~Еще не все строки переведены.~~ Ура, все строки переведены

File: locales/eo.strings [hunk 12]
```diff
@@ -569,12 +591,6 @@
...
 
-/* Paginator (deprecated) */
-
-"paginator_back" = "Reen";
-"paginator_page" = "Paĝo $1";
-"paginator_next" = "Venonta";
-
 /* Dialogs */
 
 "ok" = "OK";
```
Comment (veselcraft) [inline locales/eo.strings]: `Bone`

**Review by veselcraft — CHANGES_REQUESTED** [review]

**Pull request merged**